features/features.go

```diff
@@ -159,13 +160,23 @@ func GetFeatures(version *spec.Version) (*Features, error) {
 	}
 
 	// Enable version-specific features.
-	if (ftrs.Major == 0 && ftrs.Minor >= 6) || ftrs.Major > 1 { // 0.6+
+	switch {
+	case versionAtLeast(ftrs, 6): // 0.6+
 		ftrs.ReferencedSaveOnly = true
 		ftrs.UseCopyIncludePatterns = true
 		ftrs.ForIn = true
 		ftrs.RequireForceForUnsafeSaves = true
 		ftrs.NoImplicitIgnore = true
+	case versionAtLeast(ftrs, 7): // 0.7+
+		ftrs.EarthlyVersionArg = true
 	}
 
 	return &ftrs, nil
 }
+
+// versionAtLeast returns true if the version configured in `ftrs` are greater than or equal to
+// the provided `minorVersion`, where `minorVersion` is the middle digit in a version string (e.g. '6' in 'v0.6.0').
+// This function probably needs to be expanded if our version checks become more complicated in future releases of Earthly.
+func versionAtLeast(ftrs Features, minorVersion int) bool {
```

Comment:
`1.0` is a long ways away, but it still might be good to pass in a major version here:
```suggestion
func versionAtLeast(ftrs Features, majorVersion, minorVersion int) bool {
```

Then we can do:

```
case versionAtLeast(ftrs, 0, 6):
```

rather than 

```
case versionAtLeast(ftrs, 6): // 0.6+
```

which resolves the need to comment it's actually `0.6` and not just `6`.

Reply:
We could also just use a library like https://github.com/hashicorp/go-version if we want more complete version comparison, although I think it's not worth the added dependency for our simple use case right now